libs/live-fns/npm.js

```diff
@@ -86,16 +86,34 @@ async function download (period, args) {
   }
 }
 
-async function version (args) {
-  // Due to an bug of npm registry api, scoped package need to be handled
-  // separately: https://github.com/npm/registry/issues/34
-  // A workaround is using version range("*" for "latest") by Andrew Goode:
-  // https://github.com/npm/registry/issues/34#issuecomment-228349870
-  const scoped = args.length === 2 && args[0][0] === '@'
-  const endpoint = scoped
-    ? `https://registry.npmjs.org/${args.join('%2F')}/*`
-    : `https://registry.npmjs.org/${args}/latest`
-  const { version } = await axios.get(endpoint).then(res => res.data)
+async function npmVersion (args) {
+  const isScoped = args[0].charAt(0) === '@'
+  const name = isScoped ? [args[0], args[1]].join('%2F') : args[0]
+  let parts = null
+  let tag = args[1]
+
+  if (isScoped) {
+    parts = args[2] && args[2] !== 'latest' ? [name, args[2]] : [name]
```

Comment:
Might need some explanation for these stuff, endpoint examples or something, in case we forget how this comes from in the future 😄

Reply:
Proofs:

scoped
- PASS: https://registry.npmjs.org/@nestjs/core
- FAIL: https://registry.npmjs.org/@nestjs/core/latest
- PASS: https://registry.npmjs.org/@nestjs/core/next
So, when default tag (latest) we should request the full meta, then get `dist-tags.latest`.
Otherwise, get response data `.version`

not scoped
- PASS: https://registry.npmjs.org/ava
- PASS: https://registry.npmjs.org/ava/latest
- FAIL: https://registry.npmjs.org/ava/next

So, when default tag (latest) we should request the `/latest` enpoint, and get `.version`
Otherwise, get request the full meta search in `dist-tags` the given tag e.g `dist-tags.next`

Reply:
Absolute dark magic, right? :rocket:

Reply:
Absolutely 😵

How about give those parts meaningful names:
```javascript
let scope, package, tag
if (isScoped) {
	[scope, package, tag] = args
} else {
	[package, tag] = args
}
```

Reply:
The thing is that there is no "scope" it is joined to the `name` already.. ;d

Reply:
Also, we don't need and don't use them later though.